Formula/l/localai.rb

```diff
@@ -0,0 +1,51 @@
+class Localai < Formula
+  include Language::Python::Virtualenv
+
+  desc ":robot: The free, Open Source OpenAI alternative"
+  homepage "https://localai.io"
+  url "https://github.com/mudler/LocalAI/archive/refs/tags/v2.15.0.tar.gz"
+  sha256 "690ee0facf5450fbf9ff886ac20c6e36c68face626e2d58014cc7ccbaa330ac0"
+  license "MIT"
+
+  depends_on "abseil" => :build
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "grpc" => :build
+  depends_on "protobuf" => :build
```

Comment:
I'm sceptical that `abseil`, `grpc`, and `protobuf` are used correctly here. We don't ship static libraries, so their being `:build`-only is suspicious.

Reply:
I'm not sure, the docs don't make a distinction https://localai.io/basics/build/#example-build-on-mac 

I can remove the `:build` only and it should work properly I guess